bin/pygrb/pycbc_pygrb_results_workflow

```diff
@@ -218,6 +220,12 @@ if not set(inj_sets).issubset(eff_secs):
 inj_sets = [i.upper() for i in inj_sets]
 inj_files = labels_in_files_metadata(inj_sets, start_rundir, args.inj_files)
 
+# File instance of the veto file
+veto_file = args.veto_file
+if veto_file: 
+    veto_file = os.path.join(start_rundir, args.veto_file)
```

Comment:
Why is this necessary? Should we not assume that the workflow generator will pass a complete file path to `--veto-file`?

Reply:
Also not sure you really need the `resolve_url_to_file()` call below…

Reply:
Ah wait, this is the results workflow generator. Then it makes more sense.

Reply:
Correct, this is workflow generation stuff, so the files are getting converted to `File` instances.